taichi/backends/opengl/codegen_opengl.cpp

```diff
@@ -499,6 +499,14 @@ class KernelGen : public IRVisitor {
          const_stmt->short_name(), const_stmt->val[0].stringify());
   }
 
+  void visit(KernelReturnStmt *stmt) override {
+    used.argument = true;
+    used.int64 = true;
+    emit("_args_{}_[0] = {};", // TD: correct idx, another buf
+         "i64",//data_type_short_name(stmt->element_type()),
+         stmt->value->short_name());
```

Comment:
Too bad, still using `arg[0]` for return. We want `ret[0]` instead.
Also pls update `context`. `get_ret_int`.